src/ExadelMentorship.WebApi/Controllers/MessageController.cs

```diff
@@ -25,11 +30,18 @@ public async Task SendMail()
             await _emailSender.SendEmailAsync(message);
         }
 
-
+        [Authorize(Policy = "ApiScope")]
```

Comment:
Let's make it Role based authorization i.e. It should be authorized if the token contains "sendMail" claim.

Reply:
As stackoverflows says clients are not users, so we can't assign roles to a client.
<img width="557" alt="image" src="https://user-images.githubusercontent.com/59729680/178147121-d53f0463-8ce8-42cb-9b52-3599ef0bb679.png">